### ISSUE TITLE
Doc site is down, react.semantic-ui.com does not load

### ISSUE THREAD

**Issue by layershifter** [Semantic-Org/Semantic-UI-React#1404]
Something strange is goins with our docs.

On first open:
![_698](https://cloud.githubusercontent.com/assets/14183168/23454129/f0f68d6c-fe73-11e6-9aef-2ecf1b692d71.png)

After F5:
![_700](https://cloud.githubusercontent.com/assets/14183168/23454111/da41c5c8-fe73-11e6-8974-d023b7669d04.png)

After next F5 all works normally :confused: Seems there is problem with rawgit.

**Comment by levithomason** [comment]
Vendor:
- First load, unexpected token.  This is very odd.
- Second load, decoding failed.  Also odd.
- Third load, success.  🤷‍♂️ 

Hm, gonna have to investigate this a bit more.

**Comment by levithomason** [comment]
Per @jaylord-github in #1412, seems there is also a content encoding error coming from cdnjs. 

>Hi guys, I cannot load the page, are you having issues accessing the page also: http://react.semantic-ui.com/
>
>Error's below:
>GET http://cdnjs.cloudflare.com/ajax/libs/anchor-js/3.2.1/anchor.min.js net::ERR_CONTENT_DECODING_FAILED
>react.semantic-ui.com/:1
>GET http://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.2.9/semantic.min.css net::ERR_CONTENT_DECODING_FAILED
>react.semantic-ui.com/:28
>GET http://cdnjs.cloudflare.com/ajax/libs/babel-standalone/6.23.1/babel.min.js net::ERR_CONTENT_DECODING_FAILED
>react.semantic-ui.com/:28
>GET http://cdnjs.cloudflare.com/ajax/libs/Faker/3.1.0/faker.min.js net::ERR_CONTENT_DECODING_FAILED
>react.semantic-ui.com/:28
>GET http://cdnjs.cloudflare.com/ajax/libs/js-beautify/1.6.11/beautify-html.min.js net::ERR_CONTENT_DECODING_FAILED
>react.semantic-ui.com/:28
>GET http://cdnjs.cloudflare.com/ajax/libs/react/15.4.2/react.min.js net::ERR_CONTENT_DECODING_FAILED
>react.semantic-ui.com/:28
>GET http://cdnjs.cloudflare.com/ajax/libs/react/15.4.2/react-dom.min.js net::ERR_CONTENT_DECODING_FAILED
>react.semantic-ui.com/:28
>GET http://cdnjs.cloudflare.com/ajax/libs/react/15.4.2/react-dom-server.min.js net::ERR_CONTENT_DECODING_FAILED
>vendor.1632bf1….js:1
>Uncaught SyntaxError: Unexpected identifier
>app.d09631b….js:1
>Uncaught SyntaxError: Unexpected identifier

This is likely the same issue as #1383.  This error seems to be thrown when the HTTP headers are claiming the content is gzip encoded but it actually isn't, see [here](http://superuser.com/questions/172951/chrome-error-330-neterr-content-decoding-failed/291707).  This makes sense given the sytax errors we're seeing, the browser is likely trying to decode something that isn't encoded in the first place.

This meteor issue https://github.com/meteor/meteor/issues/7295 reports issues with certain browser extensions screwing this up as well.

Can @layershifter @jaylord-github @ben174 and @ryan4888 try turning off any browser extensions (i.e. use incognito mode) and see if this resolves the issue?  Also, please let me know which browsers this is happening in.  I'm unable to reproduce this issue still.

**Comment by levithomason** [comment]
@jlukic if you see this, any changes to the hosting recently?

**Comment by layershifter** [comment]
@levithomason These screens was captured in incognito mode. Also, there was same problems in FF.

**Comment by layershifter** [comment]
![_703](https://cloud.githubusercontent.com/assets/14183168/23498699/7df219d0-ff31-11e6-8aad-6861a75980d7.png)

**Comment by layershifter** [comment]
Another try:

![_704](https://cloud.githubusercontent.com/assets/14183168/23498775/cf5c4228-ff31-11e6-904a-be8e59ab7773.png)

**Comment by levithomason** [comment]
Are you able to post the full request and response details?  Here's what I'm getting for `vendor.js`:

### Request
```
URL:http://cdn.rawgit.com/Semantic-Org/Semantic-UI-React/gh-pages/vendor.1632bf1ec99ef59ea677.js
Request Method:GET
Status Code:200 OK
Remote Address:198.232.124.25:80
```
### Response Headers
```
Access-Control-Allow-Origin:*
Cache-Control:max-age=315569000, immutable
Connection:keep-alive
Content-Encoding:gzip
Content-Type:application/javascript;charset=utf-8
Date:Thu, 02 Mar 2017 08:19:19 GMT
ETag:W/"cdda741227eac3dc2907372b513bf965e4d18371"
RawGit-Cache-Status:BYPASS
Server:NetDNA-cache/2.2
Transfer-Encoding:chunked
Vary:Accept-Encoding
X-Cache:HIT
X-Content-Type-Options:nosniff
X-Robots-Tag:none
```
### Request Headers
```
Accept:*/*
Accept-Encoding:gzip, deflate, sdch
Accept-Language:en-US,en;q=0.8
Cache-Control:no-cache
Connection:keep-alive
Cookie:__cfduid=dd9bcc21c49055dfc43cb65f8cdace6f41475895282
DNT:1
Host:cdn.rawgit.com
Pragma:no-cache
Referer:http://react.semantic-ui.com/
User-Agent:Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/56.0.2924.87 Safari/537.36
```

**Comment by layershifter** [comment]
##  Request
```
URL:http://cdn.rawgit.com/Semantic-Org/Semantic-UI-React/gh-pages/app.d09631be433581525dac.js
Request Method:GET
Status Code:200 OK
Remote Address:198.232.124.25:80
```

## Response headers

```
Access-Control-Allow-Origin:*
Cache-Control:max-age=315569000, immutable
Connection:keep-alive
Content-Encoding:gzip
Content-Type:application/javascript;charset=utf-8
Date:Thu, 02 Mar 2017 08:24:10 GMT
ETag:W/"a125a9ba720ad6221eb253a35eaf4e9218dcebcb"
RawGit-Cache-Status:HIT
Server:NetDNA-cache/2.2
Transfer-Encoding:chunked
Vary:Accept-Encoding
X-Cache:HIT
X-Content-Type-Options:nosniff
X-Robots-Tag:none
```

## Request headers
```
Accept:*/*
Accept-Encoding:gzip, deflate, sdch
Accept-Language:ru-RU,ru;q=0.8,en-US;q=0.6,en;q=0.4
Connection:keep-alive
Host:cdn.rawgit.com
Referer:http://react.semantic-ui.com/
User-Agent:Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/54.0.2840.100 Safari/537.36
```

**Comment by layershifter** [comment]
On success load.

## Request
```
URL:http://cdn.rawgit.com/Semantic-Org/Semantic-UI-React/gh-pages/app.d09631be433581525dac.js
Request Method:GET
Status Code:200 OK
Remote Address:198.232.124.25:80
```

## Response Headers 
```
Access-Control-Allow-Origin:*
Cache-Control:max-age=315569000, immutable
Connection:keep-alive
Content-Encoding:gzip
Content-Type:application/javascript;charset=utf-8
Date:Thu, 02 Mar 2017 08:27:09 GMT
ETag:W/"a125a9ba720ad6221eb253a35eaf4e9218dcebcb"
RawGit-Cache-Status:HIT
Server:NetDNA-cache/2.2
Transfer-Encoding:chunked
Vary:Accept-Encoding
X-Cache:HIT
X-Content-Type-Options:nosniff
X-Robots-Tag:none
```

## Request Headers

```
Accept:*/*
Accept-Encoding:gzip, deflate, sdch
Accept-Language:ru-RU,ru;q=0.8,en-US;q=0.6,en;q=0.4
Cache-Control:max-age=0
Connection:keep-alive
Host:cdn.rawgit.com
Referer:http://react.semantic-ui.com/
User-Agent:Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/54.0.2840.100 Safari/537.36
```

**Comment by ryndshn** [comment]
@levithomason Like I said, the issue does not occur for me anymore. I'm not sure what caused it. However, I can say that when I was having the issue I disabled uBlock and that did not fix it. I didn't think to try incognito mode.

**Comment by levithomason** [comment]
This is helpful, I appreciate the dual update.

**Comment by levithomason** [comment]
@layershifter is this issue still persisting for you?

**Comment by layershifter** [comment]
No, everything works fine now.

**Comment by levithomason** [comment]
Ok, chalking this up to an error server side then. We changed nothing on our end. Hopefully it is resolved for good. Thanks for the prompt reply.

**Comment by layershifter** [comment]
Issue is here again.

### On fail

![_730](https://cloud.githubusercontent.com/assets/14183168/23654576/f69a3942-0339-11e7-8fd8-afd8dab6278d.png)
![_731](https://cloud.githubusercontent.com/assets/14183168/23654575/f697e6f6-0339-11e7-9de2-e173be3eb36b.png)

### On success

![_732](https://cloud.githubusercontent.com/assets/14183168/23654588/fdaa954c-0339-11e7-8f52-68f8563b5ee8.png)

**Comment by levithomason** [comment]
Does this only happen from `rawgit.com` for you or is it also happening with `cdnjs` scripts?

**Comment by layershifter** [comment]
It happens only with `rawgit.com` for me. Now, it works normally again 😕

**Comment by levithomason** [comment]
DNS is also broken, see #1429.  For now, use this temporary docs link, images will be broken but the docs work for the most part:

https://semantic-org.github.io/Semantic-UI-React/

**Comment by levithomason** [comment]
I've also updated the README.me and links on the repo home page with this information.

**Comment by levithomason** [comment]
Docs are back up.  I've removed the temporary docs warning from the README.md.

I've also updated all doc site images to use our local images, fixing #1441.  I've also updated all HTTP links to semantic-ui.com to use HTTPS so that they work with the new changes.

The DNS changes 3 days ago were highly likely the cause of the initial intermittent failures.  Once fully propagated, they resulted in complete doc site failure to resolve.

I expect everything to be stable now.
